app/src/main/java/com/example/kotlin_java_practicalss/Decision_making_Demo.java

```diff
@@ -0,0 +1,45 @@
+package com.example.kotlin_java_practicalss;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class Decision_making_Demo {
+    void if_else_eg()
+    {
+        List<String> l1= Arrays.asList("India","Australia","Paris","UK","USA");
+        System.out.println("=========If else========");
+        System.out.println("List is-->"+l1);
+        if(l1.contains("India"))
+        {
+            System.out.println("Yes! India is exist in list");
+        }
+        else
+        {
+            System.out.println("Nooo!");
+        }
+    }
+    void nested_if_eg()
```

Comment:
do not use snake case anywhere except in xml component id
for functions and variables use lowerCamelCase
for classes and enums use CamelCase